locations/PoD.json

```diff
@@ -103,8 +104,8 @@
             {
                 "name": "Compass Chest",
                 "access_rules":[
-                    "$smallKeys|pod_smallkey|2, [bombs], [bow]",
-                    "@Palace of Darkness/Map Chest, hammer, $smallKeys|pod_smallkey|1"
+                    "$smallKeys|pod_smallkey|2, [$smallKeys|pod_smallkey|4]",
+                    "@Palace of Darkness/Map Chest, Hammer, $smallKeys|pod_smallkey|1, [$smallKeys|pod_smallkey|4]"

```

Comment:
hammer needs  to be lowercase

Reply:
If that were true, Stalfos Basement wouldn't work, which I didn't change. I can lower case them all for consistency, but in this case I was trying to be consistent with what was already in the file elsewhere.

Reply:
Yeah the issue and inconsistency  came from my end. Every other item code is lowercase but i somehow missed those. 
I thought i changed them all with the last release to be consistent lowercase